src/shared/components/code-editor/openapi-to-json-schema.ts

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable no-prototype-builtins */
+
+// this file was copied from openshift/console repository - it has some util functions to convert from
```

Comment:
as I mentioned here, this file was copied from the OpenShift Console repo - it works the same as in our case, so I opted not to rewrite or modify it for now. But if you guys think that's not a good approach, I'm totally open to suggestions :)

Reply:
Which package from openshift/console has this configuration?